src/string.jl

```diff
@@ -60,8 +60,10 @@ function test_reference_string(file::File, actual::AbstractArray{<:AbstractStrin
                 answer = first(readline())
                 if answer == 'y'
                     write(path, join(actual, "\n"))
+                    warn("Please run the tests again for any changes to take effect")
+                else
+                    @test false
```

Comment:
this will still abort the test run after the testset is done, no?

Reply:
If there is no further enclosing testset, yes.
(This is why it is good to have one ottermost testset with your package name IMO).

But this is only if the answer is 'n'.
And if I've said _No, do not regenerate the test files_
then that is an actual failure

Reply:
right, good point